developer/bootstrap.rst

```diff
@@ -61,11 +61,12 @@ Ignore ``colcon-argcomplete`` and ``colcon-bash`` on Windows.
 Dependencies
 ------------
 
-Make sure the required dependencies are available:
+Make sure the dependencies are available:
 
 .. code-block:: bash
 
-    $ pip install -U EmPy
+    $ curl --output requirements.txt https://raw.githubusercontent.com/colcon/colcon.readthedocs.org/master/colcon.repos
```

Comment:
`/colcon.repos` -> `/requirements.txt`

Reply:
Doh, the one thing where I had to diverge from the instruction to get the file from this branch. Fixed in https://github.com/colcon/colcon.readthedocs.org/pull/13/commits/803a06a2481ace2b6c73921adf622a7c4a8a3f0a#diff-56a836a4f707ed1ebcb293e54154074eR68